config/jobs/kubernetes/sig-release/release-branch-jobs/1.14.yaml

```
@@ -448,6 +458,9 @@ periodics:
     labels:
       preset-service-account: "true"
       preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-release-1.14-all, sig-release-1.14-blocking, sig-node-kubelet, sig-node-cri-1.12
```

if order of dashboards matters
```suggestion
      testgrid-dashboards: sig-release-1.14-blocking, sig-release-1.14-all, sig-node-kubelet, sig-node-cri-1.12
```